### DISCUSSION TITLE
Did anyone make it to use Lama in CUDA mode with AMD GPU with Windows?

### DISCUSSION THREAD

**Discussion by noname9006** [Sanster/IOPaint#424] [General]
subj

**Comment by Sanster** [comment]
Not supported, the main bottleneck is that pytorch does not support AMD/ROCm graphics cards on Windows.  https://github.com/pytorch/pytorch/issues/106608

**Comment by Marsaalam** [comment]
![FB_IMG_1707171513829](https://github.com/Sanster/IOPaint/assets/161058018/6aed9f95-1d93-4ef3-a6c3-3a11c14fca99)

**Reply by Marsaalam** [reply]
Remove person
